### ISSUE TITLE
Unable to detect request time when using ofetch or composables derive from it

### ISSUE THREAD

**Issue by artmizu** [artmizu/nuxt-prometheus#42]
Due to this [issue in ofetch package](https://github.com/unjs/ofetch/issues/295) `@artmizu/nuxt-prometheus` cannot detect network requests which take place through useFetch or $fetch, because they both use ofetch inside. So on the /metrics page, you don't see any request time related to ofetch.

I've already tried:
1. Used different Nuxt hooks, for redefinition of global fetch. I'd try to do it before ofetch actually did it, but ofetch always did it first
2. Tried different lifehacks around nuxt context to pass request information through functions and so on. In the end, it produced memory leaks or just didn't work.
3. Try to reverse engineering ofetch to figure out the way how to overcome the issue, but for now don't see any outer way to modify its behavior, all variable is hidden inside closures, and so on

Maybe someone has an idea of how to overcome the issue? Or we'll just wait for updates from ofetch

**Comment by Gonzo17** [comment]
Nice, I've just seen that the ofetch will be patched with the next release, so this issue will likely be solved as well. :) Since I want to use this module in my new project, I can offer you to help if needed.

**Comment by artmizu** [comment]
@Gonzo17 yeah, it's nice. I will send the update as soon as ofetch team publishes their own. So stay tuned :)

**Comment by ennc0d3** [comment]
Any idea when this will be available? Thanks

**Comment by artmizu** [comment]
> Any idea when this will be available? Thanks

hi. will try to check this weekend, ofetch update is already available so hope it will work

**Comment by Rigo-m** [comment]
Let us know <3

**Comment by artmizu** [comment]
> Let us know <3

i tested the last version of nuxt in the context of compatibility with the @mswjs/interceptors package. requests through direct ofetch methods work fine and i can measure the duration, but if ofetch is wrapped in nuxt useFetch mswjs interceptors doesn't work, so right now trying to figure out why, through digging inside nuxt source code

**Comment by artmizu** [comment]
Right after this issue https://github.com/nitrojs/nitro/issues/3008 resolution, we'll finally overcome the problem

**Comment by DarkGL** [comment]
Looks like https://github.com/nitrojs/nitro/issues/3008 is closed, can we enable `enableRequestTimeMeasure` now?

**Comment by artmizu** [comment]
@DarkGL hi, thank you for the question

the team hasn't published the release yet, the last one was on November 8th 

https://github.com/nitrojs/nitro

**Comment by DarkGL** [comment]
@artmizu thank you for the explanation :)

**Comment by artmizu** [comment]
This issue has been fixed, but please be aware that you need to use the last Nuxt version to get the right measurements. The overall issue is related to the package inside Nuxt. 

https://github.com/artmizu/nuxt-prometheus/releases/tag/v2.4.1
